mat_dp_pipeline/data_sources/country_sets/create_country_to_path.py

```diff
@@ -1,32 +1,38 @@
 from pathlib import Path
 
+import pandas as pd
+
 from .country_set import CountrySet
+from .identifier import Identifier
 
 __all__ = ["CountrySets", "create_country_to_path"]
 
 CountrySets = dict[str, "CountrySet | CountrySets"]
 
 
-def create_country_to_path(country_sets: CountrySets) -> dict[str, Path]:
+def create_country_to_path(
+    country_sets: CountrySets, country_code_df: pd.DataFrame, identifier: Identifier
+) -> dict[str, Path]:
     final_dict: dict[str, list[str]] = {}
 
     def dfs(country_sets: CountrySets, prefix: list[str]):
         for k, v in country_sets.items():
             if isinstance(v, CountrySet):
-                all_countries = v.all_countries
-                for country in all_countries:
+                for country, country_name in v.all_countries(
+                    country_code_df, identifier
+                ):
                     if country in final_dict:
                         raise ValueError(
-                            f"Overlapped disjoint set detected for {country}"
+                            f"Overlapped disjoint set detected for {country} location is already {final_dict[country]}"
```

Comment:
Haha that's a funny. COmment. Not an issue, but Overlapped disjoint sounds funny ;]